package.json

```diff
@@ -31,8 +31,11 @@
     "test": "jest --color --ignoreProjects \"Gulp tasks\" --maxWorkers=50%",
     "test:screenshots": "node ./tasks/screenshot-components.mjs",
     "lint": "npm run lint:js && npm run lint:scss",
-    "lint:js": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore --max-warnings 0 \"**/*.{cjs,js,mjs}\"",
-    "lint:scss": "stylelint --cache --cache-location .cache/stylelint --cache-strategy content --color --ignore-path .gitignore --max-warnings 0 \"{app,src}/**/*.scss\""
+    "lint:js:cli": "eslint --cache --cache-location .cache/eslint --cache-strategy content --color --ignore-path .gitignore --max-warnings 0",
+    "lint:js": "npm run lint:js:cli -- \"**/*.{cjs,js,mjs}\"",
```

Comment:
@romaricpascal Sorry to be a pain, but could you move the `lint:js:cli` task below `lint:js`?

Same with `lint:scss` followed by `lint:scss:cli`

Can you tell I've been looking at [**eslint-plugin-json-files**](https://github.com/kellyselden/eslint-plugin-json-files#readme) for `package.json` linting? 🤦‍♂️